contact_map/version.py

```diff
@@ -0,0 +1,141 @@
+# This file vendored from Autorelease
+import os
+import subprocess
+
+try:
+    from configparser import ConfigParser, NoSectionError, NoOptionError
+except ImportError:
+    # py2
```

Comment:
small question, do we really want to keep support for py2 around (EOL is approaching)

Reply:
Vendored code. Autorelease supports 2.7 because it is used (well, about to be used) in other projects that do support 2.7 (like the upcoming 1.x releases of OPS).

Having the support there makes no difference even if (when) Contact Map Explorer goes all-Py3. It just won't be used. Since it is vendored, it doesn't count as something that we have to maintain here. (I have to maintain it *elsewhere*, not here!)

This does get back to the conceptual thing that's kind of weird here: `setup.py` and `version.py` should be considered read-only (as much as possible). If they're changed, the changes should happen in Autorelease.

Normally I'm very opposed to vendoring code, because of exactly this problem of synchronizing changes. Better to make it a requirement. However, when the `setup.py` itself is what is vendored, you end up in the trade-offs that come with PEP517 (including losing developer installs). Vendoring seemed like the lesser evil.

Reply:
> Vendoring seemed like the lesser evil.

I agree with you, and could not think of a better way to resolve this at the moment. Feel free to merge on green